config/aws_idc.tf

```
@@ -8,14 +8,14 @@ resource "commonfate_access_workflow" "aws" {
 resource "commonfate_aws_account_selector" "select_all" {
   id                  = "select_all_aws"
   name                = "Select All AWS"
-  aws_organization_id = "<Your AWS Organization ID>"
+  aws_organization_id = <Your external AWS Organization ID>
```

'external' doesn't make sense here I think, I think we should revert to just "Your AWS Organization ID". The organization ID is for their own organization - it isn't external to them. (Although I see what you were doing here in trying to make it clearer). For a BYOC deploy CF is deployed into the same AWS organization so there is no internal/external difference.

```suggestion
  aws_organization_id = <Your AWS Organization ID>
```